manage.py

```diff
@@ -311,6 +313,27 @@ def update_node_packages(project_config):
     except Exception as e:
         print("An error occurred during Node.js package update: " + str(e))
 
+def toggle_maintenance_mode(project_config):
+    project_path = project_config['path']
```

Comment:
Can we also check for existence of params? The exception output is a pain to trap.

Reply:
@apschultz did one better and cleaned up the conditionals in favor of a dedicated config_check method in https://github.com/ClanAODDev/deploy/pull/6/commits/a365d84e74f697346eb51c53f3980808211c55a5